andela/savage_rangers-ah-frontend
# #166240794 Users should be able to view notifications

#### What does this PR do?
Allow user to view notifications in real time

#### Description of Task to be completed?
* Design the UI
* Consume endpoints 
* Write tests

#### How should this be manually tested?
* Login to Authors haven the check your profile picture in the top right side of the home page.

#### Any background context you want to provide?
* There is a socket created to listen to events broadcasted from the backend

#### Screenshots (if appropriate)
<img width="557" alt="Screen Shot 2019-08-27 at 17 00 15" src="https://user-images.githubusercontent.com/47884449/63782772-3182bf00-c8ec-11e9-940b-1d63a461291b.png">
